src/util/request.js

```diff
@@ -139,7 +139,17 @@ const sendRequest = async ({
     histTimerEnd({ success: true, source, destination, method })
     return response
   } catch (error) {
-    logger.warn('error in request.sendRequest:', error)
+    logger.error('error in request.sendRequest:', {
```

Comment:
I don't think we need to log it with `error` severity, coz it's an infrastructural level. The domain level should decide how to handle http error and whether it worse logging an error or just handle it.  E.g. if we have 404, I don't  see any reason to log it as error (in most cases).

Reply:
Let's refine the level based on observations. We could base it on HTTP status codes.